extensions/wikia/AdEngine/js/spec/provider/gpt/helper.spec.js

```diff
@@ -203,6 +211,7 @@ describe('ext.wikia.adEngine.provider.gpt.helper', function () {
 			return true;
 		};
 
+		console.log(mocks.adContext.getContext().targeting);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this logging statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AS2228)